File: mycollab-web/src/main/scala/com/mycollab/module/user/accountsettings/view/events/SetupEvent.scala
package com.mycollab.module.user.accountsettings.view.events

import com.mycollab.events.ApplicationEvent

/**
 * @author MyCollab Ltd
 * @since 5.0.5
 */
object SetupEvent {

    class GotoSetupPage(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}

}

File: mycollab-web/src/main/java/com/mycollab/module/user/accountsettings/view/event/SetupEvent.kt
package com.mycollab.module.user.accountsettings.view.event

import com.mycollab.events.ApplicationEvent

/**
 * @author MyCollab Ltd
 * @since 6.0.0
 */
class SetupEvent {
    class GotoSetupPage(source: Any, data: Any?) : ApplicationEvent(source, data)
}